package.json

```diff
@@ -13,7 +13,8 @@
     "test": "vitest run",
     "test:dev": "vitest",
     "ci": "pnpm run lint && pnpm run test && pnpm run build",
-    "release": "pnpm run ci && changeset publish"
+    "release": "pnpm run ci && changeset publish",
+    "dev:watch": "tsup src/index.ts --format cjs,esm --dts --watch"
```

Comment:
This allows us to code side to side with a console app to check out possible internal issues such as this PR's case where we had some mismatch between the behaviour when using the lib and the one when we ran the tests within the lib itself